book/src/keymap.md

```diff
@@ -320,10 +320,12 @@ Displays documentation for item under cursor. Remapping currently not supported.
 
 Displays documentation for the selected completion item. Remapping currently not supported.
 
-| Key                         | Description    |
-| ----                        | -----------    |
-| `Shift-Tab`, `Ctrl-p`, `Up` | Previous entry |
-| `Tab`, `Ctrl-n`, `Down`     | Next entry     |
+| Key                         | Description                      |
+| ----                        | -----------                      |
+| `Shift-Tab`, `Ctrl-p`, `Up` | Previous entry                   |
+| `Tab`, `Ctrl-n`, `Down`     | Next entry                       |
+| `Escape`                    | Close menu and accept completion |
+| `Ctrl-c`                    | Close menu and reject completion |
```

Comment:
The only other one that's special cased I believe is `Enter` which just accepts the completion item